templates/c_profile.html

```diff
@@ -5,103 +5,148 @@
     <title>Platform</title>
     <meta name="viewport" content=
         "width=device-width,initial-scale=1">
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/c_profile.css') }}">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link rel="stylesheet" href="c_profile.css">
   </head>
   
   <body>
       <div class="background">
         <div class="nav">
-          <a href="/quiz">Take Quiz</a>
-          <a href="/toolkit">View Toolkit</a>
-          <a href="/profile"><strong>Your Profile</strong></a>
-          <a href="/home" class="active">Home</a>
+          <div class="profile-dropdown-menu">
+            <a class="active" id="navmenu"><strong>Your Profile</strong></a>
+            <div class="profile-dropdown-content">
+              <a href=c_profile.html>Edit Profile</a>
+              <a href=c_login.html>Logout</a>
+            </div>
+          </div>
+          <a href=c_quiz_start.html id="navmenu">Take Quiz</a>

```

Comment:
Cannot add hyperlink to another HTML file directly. Please use a corresponding route you have defined in program1.py.